corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -376,9 +376,16 @@ hqDefine("cloudcare/js/formplayer/app", function () {
 
         window.addEventListener(
             'beforeprint', function () {
-                let lastPanelElement = Utils.getLastClassEl('.panel.panel-default, .q.form-group');
-                if (lastPanelElement) {lastPanelElement.classList.add('last');}
-            });
+                let lastPanelElement = $('.panel.panel-default, .q.form-group').last();
```

Comment:
Seeing the new version of this code, it could be even more concise: `$('.panel.panel-default, .q.form-group').addClass("last");` (it won't do anything if no element is found).

Reply:
Always love to hear feedback on more conciseness

The if check was a remnant of when I was using `lastPanelElement.classList`, which would throw an error if `lastPanelElement` is empty.